.pre-commit-config.yaml

```
@@ -19,13 +19,13 @@ repos:
   - id: black
 
 - repo: https://github.com/astral-sh/ruff-pre-commit
-  rev: v0.0.275
+  rev: v0.0.276
   hooks:
   - id: ruff
     args: ["--fix", "--show-fixes"]
 
 - repo: https://github.com/asottile/setup-cfg-fmt
-  rev: v2.3.0
+  rev: v2.4.0
   hooks:
   - id: setup-cfg-fmt
     args: [--include-version-classifiers, --max-py-version=3.11]
```

```suggestion
    args: [--include-version-classifiers, --min-py-version=3.7, --max-py-version=3.11]
```

(Wouldn't have been needed with #1526)